private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -228,7 +228,7 @@ export class TemplateServiceClient {
       version: version || "1.0"
     };
 
-    let templateName = name? name : "Untitled Template";
+    let templateName = name ? name : "Untitled Template";
```

Comment:
You don't have to, but you can make this `name || "Untitled Template"`

Reply:
Done.